### ISSUE TITLE
Tests in Bun adapter should be written with the Bun test harness

### ISSUE THREAD

**Issue by blaine-arcjet** [arcjet/arcjet-js#769]
We currently don't test adapters. When we add them to the Bun adapter, we should write them with the Bun test harness instead of Jest.

**Comment by blaine-arcjet** [comment]
Bun 1.2.6 added support for `node:test` that runs on top of the Bun test runner.

See https://bun.sh/blog/bun-v1.2.6#initial-support-for-node-test

**Comment by wooorm-arcjet** [comment]
I guess this can be closed, as it’s about using `node:test` everywhere. Or, the issue could be renamed `Add tests to adapters`

**Comment by blaine-arcjet** [comment]
You are correct that adapters don't currently have tests. We should change the title.
